### PR TITLE
Include status in client views

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

File: stats/HTTP.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Buckets for distributions in default views are as follows:
 
 ### Measures
 
-Client stats are recorded for each individual HTTP request, including for each individual redirect (followed or not).
+Client stats are recorded for each individual HTTP request, including for each individual redirect (followed or not). All stats are recorded after request processing (usually after the response body has been fully read).
```
Comment (SergeyKanzhelev) [inline stats/HTTP.md]: comment on inclusion/exclusion of body read callbacks into the duration would be helpful. Since you are on it =)

Comment (ramonza) [reply stats/HTTP.md]: What is a body read callback?

Comment (SergeyKanzhelev) [reply stats/HTTP.md]: I meant do you include time on parsing of http body or not

Comment (ramonza) [reply stats/HTTP.md]: It depends on whether such processing happens before the body has been fully read. For example, in a streaming system you would process input as it is being read and then you would include this processing in the stats. But if you read it all into a temporary buffer and then process it then you would not include this time.

I think this is already entailed by the statement "after the response body has been fully read" so I don't want to add anything about *how* the response body is processed as this is likely to be language-specific.

**Pull request merged**